src/dijkstra_spath.jl

```diff
@@ -247,3 +247,37 @@ end
 dijkstra_shortest_paths{V}(
     graph::AbstractGraph{V}, s::V
 ) = dijkstra_shortest_paths(graph, ones(num_vertices(graph)), s)
+
+function dijkstra_shortest_paths_explicit{V}(g::AbstractGraph{V},source::V, all...)
+    state = dijkstra_shortest_paths(g, source, all...)
+    allvertices = g.vertices
+    patharr = Array(Vector{V},0)
+    parents = state.parents
+    dists = state.dists
+    for i in 1:length(parents)
+        path = V[]
+        currvertex = allvertices[i]
+        connected = isdefined(parents,i) && (parents[i] in allvertices)
```

Comment:
this is going to be pretty inefficient since `allvertices` is usually quite large; see the workaround proposed in https://github.com/JuliaLang/Graphs.jl/issues/140